[jxrlib] build error

Package: jxrlib:x64-linux@2019.10.9#6

**Host Environment**

- Host: x64-linux
- Compiler: GNU 14.1.1
-    vcpkg-tool version: 2024-04-23-d6945642ee5c3076addd1a42c331bbf4cfc97457
    vcpkg-scripts version: 6f1ddd6b6 2024-06-07 (4 hours ago)

**To Reproduce**

`vcpkg install jxrlib`

**Failure logs**

```
-- Downloading https://github.com/4creators/jxrlib/archive/f7521879862b9085318e814c6157490dd9dbbdb4.tar.gz -> 4creators-jxrlib-f7521879862b9085318e814c6157490dd9dbbdb4.tar.gz...
-- Extracting source /home/pattakosn/github.com/vcpkg/downloads/4creators-jxrlib-f7521879862b9085318e814c6157490dd9dbbdb4.tar.gz
-- Applying patch guiddef.patch
-- Applying patch fix-mingw.patch
-- Using source at /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean
-- Found external ninja('1.12.1').
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/cmake --build . --config Debug --target install -- -v -j17
    Working Directory: /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/x64-linux-dbg
    See logs for more information:
      /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/jxrlib/portfile.cmake:20 (vcpkg_cmake_install)
  scripts/ports.cmake:175 (include)



```

<details><summary>/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/install-x64-linux-dbg-out.log</summary>

```
Change Dir: '/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/x64-linux-dbg'

Run Build Command(s): /usr/bin/ninja -v -v -j17 install
[1/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/encode/encode.c.o -MF CMakeFiles/jpegxr.dir/image/encode/encode.c.o.d -o CMakeFiles/jpegxr.dir/image/encode/encode.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/encode/encode.c
[2/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/decode.c.o -MF CMakeFiles/jpegxr.dir/image/decode/decode.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/decode.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/decode.c
[3/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/image.c.o -MF CMakeFiles/jpegxr.dir/image/sys/image.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/image.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/image.c
[4/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/strTransform.c.o -MF CMakeFiles/jpegxr.dir/image/sys/strTransform.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/strTransform.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strTransform.c
[5/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/adapthuff.c.o -MF CMakeFiles/jpegxr.dir/image/sys/adapthuff.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/adapthuff.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/adapthuff.c
[6/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/postprocess.c.o -MF CMakeFiles/jpegxr.dir/image/decode/postprocess.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/postprocess.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/postprocess.c
[7/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/strPredQuant.c.o -MF CMakeFiles/jpegxr.dir/image/sys/strPredQuant.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/strPredQuant.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strPredQuant.c
[8/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/strPredQuantDec.c.o -MF CMakeFiles/jpegxr.dir/image/decode/strPredQuantDec.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/strPredQuantDec.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/strPredQuantDec.c
[9/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/perfTimerANSI.c.o -MF CMakeFiles/jpegxr.dir/image/sys/perfTimerANSI.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/perfTimerANSI.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/perfTimerANSI.c
[10/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/encode/strPredQuantEnc.c.o -MF CMakeFiles/jpegxr.dir/image/encode/strPredQuantEnc.c.o.d -o CMakeFiles/jpegxr.dir/image/encode/strPredQuantEnc.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/encode/strPredQuantEnc.c
[11/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c
FAILED: CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o 
/usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTest.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c
In file included from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRMeta.h:30,
                 from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlue.h:34,
                 from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.h:34,
                 from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:29:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c: In function ‘PKTestFactory_CreateDecoderFromFile’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:211:33: error: passing argument 2 of ‘GetTestDecodeIID’ from incompatible pointer type [-Wincompatible-pointer-types]
  211 |     Call(GetTestDecodeIID(pExt, &pIID));
      |                                 ^~~~~
      |                                 |
      |                                 PKIID ** {aka unsigned int **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:211:5: note: in expansion of macro ‘Call’
  211 |     Call(GetTestDecodeIID(pExt, &pIID));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:108:55: note: expected ‘const PKIID **’ {aka ‘const unsigned int **’} but argument is of type ‘PKIID **’ {aka ‘unsigned int **’}
  108 | ERR GetTestDecodeIID(const char* szExt, const PKIID** ppIID)
      |                                         ~~~~~~~~~~~~~~^~~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:217:42: error: passing argument 2 of ‘PKTestFactory_CreateCodec’ from incompatible pointer type [-Wincompatible-pointer-types]
  217 |     Call(PKTestFactory_CreateCodec(pIID, ppDecoder));
      |                                          ^~~~~~~~~
      |                                          |
      |                                          PKImageDecode ** {aka struct tagPKImageDecode **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:217:5: note: in expansion of macro ‘Call’
  217 |     Call(PKTestFactory_CreateCodec(pIID, ppDecoder));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:125:56: note: expected ‘void **’ but argument is of type ‘PKImageDecode **’ {aka ‘struct tagPKImageDecode **’}
  125 | ERR PKTestFactory_CreateCodec(const PKIID* iid, void** ppv)
      |                                                 ~~~~~~~^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c: In function ‘PKCreateTestFactory’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:235:18: error: passing argument 1 of ‘PKAlloc’ from incompatible pointer type [-Wincompatible-pointer-types]
  235 |     Call(PKAlloc(ppCFactory, sizeof(**ppCFactory)));
      |                  ^~~~~~~~~~
      |                  |
      |                  PKCodecFactory ** {aka struct tagPKCodecFactory **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:235:5: note: in expansion of macro ‘Call’
  235 |     Call(PKAlloc(ppCFactory, sizeof(**ppCFactory)));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlue.h:291:20: note: expected ‘void **’ but argument is of type ‘PKCodecFactory **’ {aka ‘struct tagPKCodecFactory **’}
  291 | ERR PKAlloc(void** ppv, size_t cb);
      |             ~~~~~~~^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c: In function ‘PKTestDecode_Release’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:290:19: error: passing argument 1 of ‘PKFree’ from incompatible pointer type [-Wincompatible-pointer-types]
  290 |     return PKFree(ppID);
      |                   ^~~~
      |                   |
      |                   PKTestDecode ** {aka struct tagPKTestDecode **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlue.h:292:19: note: expected ‘void **’ but argument is of type ‘PKTestDecode **’ {aka ‘struct tagPKTestDecode **’}
  292 | ERR PKFree(void** ppv);
      |            ~~~~~~~^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c: In function ‘PKTestDecode_Create’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:299:18: error: passing argument 1 of ‘PKAlloc’ from incompatible pointer type [-Wincompatible-pointer-types]
  299 |     Call(PKAlloc(ppID, sizeof(**ppID)));
      |                  ^~~~
      |                  |
      |                  PKTestDecode ** {aka struct tagPKTestDecode **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:299:5: note: in expansion of macro ‘Call’
  299 |     Call(PKAlloc(ppID, sizeof(**ppID)));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlue.h:291:20: note: expected ‘void **’ but argument is of type ‘PKTestDecode **’ {aka ‘struct tagPKTestDecode **’}
  291 | ERR PKAlloc(void** ppv, size_t cb);
      |             ~~~~~~~^~~
[12/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/segdec.c.o -MF CMakeFiles/jpegxr.dir/image/decode/segdec.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/segdec.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/segdec.c
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/segdec.c: In function ‘_load4’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/segdec.c:66:12: warning: implicit declaration of function ‘_byteswap_ulong’ [-Wimplicit-function-declaration]
   66 |     return _byteswap_ulong(*(U32*)pv);
      |            ^~~~~~~~~~~~~~~
[13/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/encode/segenc.c.o -MF CMakeFiles/jpegxr.dir/image/encode/segenc.c.o.d -o CMakeFiles/jpegxr.dir/image/encode/segenc.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/encode/segenc.c
[14/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestBmp.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestBmp.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestBmp.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestBmp.c
[15/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/sys/strcodec.c.o -MF CMakeFiles/jpegxr.dir/image/sys/strcodec.c.o.d -o CMakeFiles/jpegxr.dir/image/sys/strcodec.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strcodec.c
[16/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/encode/strFwdTransform.c.o -MF CMakeFiles/jpegxr.dir/image/encode/strFwdTransform.c.o.d -o CMakeFiles/jpegxr.dir/image/encode/strFwdTransform.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/encode/strFwdTransform.c
[17/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/JXRTranscode.c.o -MF CMakeFiles/jpegxr.dir/image/decode/JXRTranscode.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/JXRTranscode.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/JXRTranscode.c
[18/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c
FAILED: CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o 
/usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestTif.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c
In file included from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strcodec.h:32,
                 from /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c:31:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c: In function ‘PKImageDecode_Release_TIF’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c:912:18: error: passing argument 1 of ‘WMPFree’ from incompatible pointer type [-Wincompatible-pointer-types]
  912 |     Call(WMPFree(&pID->EXT.TIF.uStripOffsets));
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                  |
      |                  U32 ** {aka unsigned int **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c:912:5: note: in expansion of macro ‘Call’
  912 |     Call(WMPFree(&pID->EXT.TIF.uStripOffsets));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strcodec.h:479:20: note: expected ‘void **’ but argument is of type ‘U32 **’ {aka ‘unsigned int **’}
  479 | ERR WMPFree(void** ppv);
      |             ~~~~~~~^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c:913:18: error: passing argument 1 of ‘WMPFree’ from incompatible pointer type [-Wincompatible-pointer-types]
  913 |     Call(WMPFree(&pID->EXT.TIF.uStripByteCounts));
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                  |
      |                  U32 ** {aka unsigned int **}
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/windowsmediaphoto.h:245:23: note: in definition of macro ‘Failed’
  245 | #define Failed(err) ((err)<0)
      |                       ^~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestTif.c:913:5: note: in expansion of macro ‘Call’
  913 |     Call(WMPFree(&pID->EXT.TIF.uStripByteCounts));
      |     ^~~~
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys/strcodec.h:479:20: note: expected ‘void **’ but argument is of type ‘U32 **’ {aka ‘unsigned int **’}
  479 | ERR WMPFree(void** ppv);
      |             ~~~~~~~^~~
[19/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestHdr.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestHdr.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestHdr.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestHdr.c
[20/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestPnm.c.o -MF CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestPnm.c.o.d -o CMakeFiles/jxrglue.dir/jxrtestlib/JXRTestPnm.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTestPnm.c
[21/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlue.c.o -MF CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlue.c.o.d -o CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlue.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlue.c
[22/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrgluelib/JXRMeta.c.o -MF CMakeFiles/jxrglue.dir/jxrgluelib/JXRMeta.c.o.d -o CMakeFiles/jxrglue.dir/jxrgluelib/JXRMeta.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRMeta.c
[23/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/strInvTransform.c.o -MF CMakeFiles/jpegxr.dir/image/decode/strInvTransform.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/strInvTransform.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/strInvTransform.c
[24/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/encode/strenc.c.o -MF CMakeFiles/jpegxr.dir/image/encode/strenc.c.o.d -o CMakeFiles/jpegxr.dir/image/encode/strenc.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/encode/strenc.c
[25/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrgluelib/JXRGluePFC.c.o -MF CMakeFiles/jxrglue.dir/jxrgluelib/JXRGluePFC.c.o.d -o CMakeFiles/jxrglue.dir/jxrgluelib/JXRGluePFC.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGluePFC.c
[26/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlueJxr.c.o -MF CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlueJxr.c.o.d -o CMakeFiles/jxrglue.dir/jxrgluelib/JXRGlueJxr.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlueJxr.c
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlueJxr.c: In function ‘CalcMetadataSizeLPWSTR’:
/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib/JXRGlueJxr.c:66:48: warning: implicit declaration of function ‘wcslen’ [-Wimplicit-function-declaration]
   66 |         U32 uiCBWithNull = sizeof(U16) * ((U32)wcslen((wchar_t *) var.VT.pwszVal) + 1); // +1 for NULL term;
      |                                                ^~~~~~
[27/31] /usr/bin/cc -DDISABLE_PERF_MEASUREMENT -D__ANSI__ -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/common/include -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/sys -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrgluelib -I/home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib -fPIC -g -Wno-error=implicit-function-declaration -Wno-endif-labels -MD -MT CMakeFiles/jpegxr.dir/image/decode/strdec.c.o -MF CMakeFiles/jpegxr.dir/image/decode/strdec.c.o.d -o CMakeFiles/jpegxr.dir/image/decode/strdec.c.o -c /home/pattakosn/github.com/vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/image/decode/strdec.c
ninja: build stopped: subcommand failed.
```
</details>

Reproduced with `g++-14` in `ubuntu:latest`:
```log
vcpkg/buildtrees/jxrlib/src/0dd9dbbdb4-ff67a15ca3.clean/jxrtestlib/JXRTest.c:211:33: error: passing argument 2 of ‘GetTestDecodeIID’ from incompatible pointer type [-Wincompatible-pointer-types]
```

--- a/ports/jxrlib/CMakeLists.txt
+++ b/ports/jxrlib/CMakeLists.txt
@@ -22,7 +22,11 @@ if(NOT WIN32)
   add_definitions(-D__ANSI__)
 endif()
 if(NOT MSVC)
-  add_compile_options(-Wno-error=implicit-function-declaration -Wno-endif-labels)
+  add_compile_options(
+    -Wno-error=implicit-function-declaration
+    -Wno-endif-labels
+    -Wno-incompatible-pointer-types             # https://gcc.gnu.org/gcc-14/porting_to.html#incompatible-pointer-types
+  )
 endif()
 
 include(TestBigEndian)

--- a/ports/jxrlib/vcpkg.json
+++ b/ports/jxrlib/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "jxrlib",
   "version": "2019.10.9",
-  "port-version": 6,
+  "port-version": 7,
   "description": "Open source implementation of the jpegxr image format standard.",
   "homepage": "https://github.com/4creators/jxrlib",
   "license": "BSD-2-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3826,7 +3826,7 @@
     },
     "jxrlib": {
       "baseline": "2019.10.9",
-      "port-version": 6
+      "port-version": 7
     },
     "kaitai-struct-cpp-stl-runtime": {
       "baseline": "0.10.1",

--- a/versions/j-/jxrlib.json
+++ b/versions/j-/jxrlib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "4d6cb4015844afea80f96aa9cd6c43fb2bdd1cc1",
+      "version": "2019.10.9",
+      "port-version": 7
+    },
+    {
       "git-tree": "3a114c06061c6e5a6e1dcc40eaadb21cecb7ac99",
       "version": "2019.10.9",
       "port-version": 6

